### PR TITLE
Fix datadog statsd ipv6

### PR DESCRIPTION
**Pull request by yunfeng-scale**
Last gift @phil-scale gave us :)

without https://github.com/DataDog/datadogpy/pull/783 datadog statsd won't work

### CODE REVIEW

**Review by seanshi-scale — COMMENTED** [review]
oops I also have https://github.com/scaleapi/models/pull/7596

**Comment by seanshi-scale** [comment]
idk if my thing works though

**Review by seanshi-scale — APPROVED** [review]

**Comment by seanshi-scale** [comment]
Internal changes have been made in a separate pr, which should fix things. Will close this for now

File: model-engine/requirements.in [hunk 1]
```diff
@@ -14,7 +14,7 @@ cloudpickle==2.1.0
 croniter==1.4.1
 dataclasses-json>=0.5.7
 datadog-api-client==2.11.0
-datadog~=0.46.0
+git+https://github.com/DataDog/datadogpy.git@a764cb054d9a3089a3a23779be8d64f18346aeb9
```
Comment (yixu34) [inline model-engine/requirements.in:17]: I thought @louisbarrett wanted us in these situations to use our own fork? Or is this ok?

Comment (yunfeng-scale) [reply model-engine/requirements.in:17]: this PR is not merged though, we're using our fork of datadogpy for this right now

**Pull request closed without merging**